cfme/services/myservice/ssui.py

```diff
@@ -76,7 +132,62 @@ def update(self, updates):
         lambda: view.is_displayed, delay=15, num_sec=300,
         message="waiting for view to be displayed"
     )
-    # TODO - implement notifications and then assert.
+    # TODO - remove sleep when BZ 1496233 is fixed
+    time.sleep(10)
```

Comment:
So we could put a Bug check in there. So that we physically check if the bug is still open, then we wait, otherwise we don't. I'm happy if we don't do this, but I would like a GitHub Issue open.